mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.tmpl

```diff
@@ -1086,7 +1086,9 @@ func resourceStorageBucketDelete(d *schema.ResourceData, meta interface{}) error
 
 		cacheList, cacheListErr := getAnywhereCacheListResult(config, bucket)
 		if cacheListErr != nil {
-			return cacheListErr
+			// If we get any error, try deleting the bucket anyway in case it's empty
+			// This would help our customers to avoid requiring extra storage.anywhereCaches.list permission.
+			break
```

Comment:
I not sure I agree with this implementation. The user should not be able to delete with-out force_destory in this case as we do not know the state of the buckets content. See line 1112

Reply:
reasonable. thanks!